Sambamba/0.6.8/Index.nf

```diff
@@ -0,0 +1,20 @@
+process Index {
+  tag {"SAMBAMBA_index ${sample_id}"}
+  label 'SAMBAMBA_index_0_6_8'
+  clusterOptions = workflow.profile == "sge" ? "-l h_vmem=${params.index_mem}" : ""
+  shell = ['/bin/bash', '-euo', 'pipefail']
+
+  input:
+    tuple sample_id, file(bams)
+
+  output:
+    tuple sample_id, file("${bams}.bai")
+
+  script:
+  """
+  sambamba index -t ${task.cpus} $bams ${bams}.bai
+  """
```

Comment:
Does this work for a bam or multiple bams?

Reply:
This is indeed a single bam/bai per tuple